pyproject.toml

```diff
@@ -31,7 +31,7 @@ classifiers = [
     "Typing :: Typed",
 ]
 
-dependencies = ["tabulate"]
+dependencies = ["tabulate", "fsspec"]
```

Comment:
In my opinion, we should make this optional (i.e. by importing `fsspec` only locally in the artifact load method), and facilitate local loading with builtin methods.

Reply:
Agree